src/parse/errors.rs

```diff
@@ -850,7 +850,8 @@ impl Error {
             c.none("'")?;
         }
 
-        c.none(&format!(": {}", err))?;
+        // need a newline here - no other message follows this.
+        c.none(&format!(": {}\n", err))?;
```

Comment:
What if `err` already ends with LF? I'd rather avoid multiple `\n`s here.

Reply:
THe `err` doesn't end with `\n` because we are building it. But yeah, it should be checked for all the usages.

Reply:
Decided to switch this to add `try_help()`, for consistency with the other cases in this file. Unit test will ensure that whatever is printed to the console has a newline